BuyPolish/Pola/Manager/BarcodeDetector.swift

```diff
@@ -0,0 +1,42 @@
+//  BarcodeDetector.swift
```

Comment:
Jak jesteś bardzo przywiązany do tego komentarza to ok. Ale ja chętnie bym się pozbył tych wszystkich automatycznych komentarzy z nagłówka.